src/werewolf/chat/game.py

```diff
@@ -40,6 +46,10 @@ def handle_special(self, type, player1=None, player2=None):
 
     # generate winning team based on voted out
     def get_winner(self):
```

Comment:
rather make this a function in role_manager instead, if we have to handle other roles later